contracts/BSCValidatorSet.sol

```diff
@@ -312,25 +346,49 @@ contract BSCValidatorSet is IBSCValidatorSet, System, IParamSubscriber, IApplica
     return CODE_OK;
   }
 
-  function getValidators()external view returns(address[] memory) {
+  function getValidators() public view returns (address[] memory) {
     uint n = currentValidatorSet.length;
     uint valid = 0;
     for (uint i = 0;i<n;i++) {
-      if (!currentValidatorSet[i].jailed) {
+      if (isWorkingValidator(i)) {
         valid ++;
       }
     }
     address[] memory consensusAddrs = new address[](valid);
     valid = 0;
     for (uint i = 0;i<n;i++) {
```

Comment:
format issue `uint i = 0;i<n;i++`, as well as line 352

Reply:
This is old code, and I try not to change the old code if it is not necessary